.github/workflows/all_plugins.yaml

```diff
@@ -31,6 +31,19 @@ jobs:
       - uses: bluefireteam/melos-action@dd3c344d731938d2ab2567a261f54a19a68b5f6a
       - name: 'Run Analyze'
         run: melos run analyze
+
+  # Separated from analyse as pubspec_override file is not being taken into account when running flutter pub publish --dry-run
+  # Need to run on Flutter 3.7.0
+  pub_dry_run:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+      - uses: subosito/flutter-action@dbf1fa04f4d2e52c33185153d06cdb5443aa189d
+        with:
+          flutter-version: '3.7.0'
```

Comment:
I might have misexplained the issue, but the Flutter version should be 3.3.0 if we want this to work. You can check locally by changing a platform interface.
Probably should have a TODO here to remember to change when it will be fixed

Reply:
You're right, but it fails if I use 3.3.0 as well 🤔 due to version constraints: https://github.com/firebase/flutterfire/actions/runs/4223880879/jobs/7334133477